akka-docs/src/main/paradox/stream/operators/ActorSink/actorRef.md

```diff
@@ -1,6 +1,6 @@
 # ActorSink.actorRef
 
-Sends the elements of the stream to the given @java[`ActorRef<T>`]@scala[`ActorRef[T]`], without considering backpressure.
+Sends the elements of the stream to the given @java[`ActorRef<T>`]@scala[`ActorRef[T]`] of the new actors API, without considering backpressure.
```

Comment:
I don't like this so much, how about instead something like we have on other pages for things that exist for both typed and classic, "This covers interaction with the new actor API, for the classic actor API see [that other operator]"?

Reply:
I added all to the "See also".
This sentence gets picked up into the indexing pages, I wanted it to be more obvious there which API we're talking about.

Reply:
ok